docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+
+ --- 
+## \<Preload/Auto\> 
+ (D9 only) Preload the Scene Master preset state (loads the LTP values of non-illuminated fixtures so they don't visibly move when the state is faded in). See [Scene Master](../running-the-show/playback-controls.md/#scene-master).
+
+ --- 
+## \<Prev Step\> 
+ Moves to the previous cue in the connected cue list using programmed fade times. Labelled \<Prev Cue\> or \<Cue -\> on some consoles. See [Running a cue list](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Record\> 
+ Enter Record mode to record playbacks or palettes. See [Creating a cue](../cues/creating-a-cue.md/#creating-a-cue) and [Storing a palette](../palettes/creating-palettes.md/#storing-a-palette)
+
+ --- 
+## \<Record Step\> 
+ Record the programmer into the current active cue in the connected cue list. See [Editing a cue list while recording](../cue-lists/editing-cue-lists.md/#editing-a-cue-list-while-recording)
+
+ --- 
+## \<Release\> 
+ Used to override the LTP behaviour of attributes when a playback is killed. See [Release](../cues/cue-playback.md/#release)
+
+ --- 
+## \<Rem Dim\> 
+ Turns off all unselected fixtures. Not fitted on all consoles, can also be accessed by \<Avo\> + \<All\>. See [Turn Off Unselected Fixtures](../controlling-fixtures.md/#turn-off-unselected-fixtures-remainder-dim)
+
+ --- 
+## \<Reset\> 
+ (D9 only) Clear the preset state of the Scene Master. See [Scene Master](../running-the-show/playback-controls.md/#scene-master).
+
+ --- 
+## \<Review\> 
+ Re-runs the fade times of the current live cue to allow you to see timing changes. See [Edit Times of a Running Cue List](../cue-lists/editing-cue-lists.md/#edit-times-of-a-running-cue-list)
+
+ --- 
+## \<Right\> 
+ (D9 / ST only) Right mouse button when trackball in mouse mode.
+
+ --- 
+## \<Select If\> 
+ Selects all fixtures which have intensity above zero. See [Select If](../controlling-fixtures.md/#selecting-fixtures-which-are-in-a-playback).
+
+ --- 
+## \<Set\> 
+ This button has become the \<Time\> button.
+
+ --- 
+## \<Shape\> 
+ Selects the Shape/FX menu to run shapes, key frame shapes or pixel map effects. See [Shapes and Effects](../effects.md).
+
+ --- 
+## \<Size/Position\> 
+ Moves the currently selected workspace window around the different size and position options. If \<Avo\> held also, moves the window to the other screen.
+
+ --- 
+## \<Snap\> 
+ Enables Snap mode which makes \<Prev Step\> / \<Next Step\> buttons snap through cues without using the programmed fade times. See [Running a cue list](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Special\> 
+ Selects the Special attribute group so the wheels control fixture-specific functions like modes and macros. Also used for creating masks. See [Changing Attributes using the Wheels](../controlling-fixtures/changing-fixture-attributes.md/#changing-attributes-using-the-wheels).
+
+ --- 
+## \<Stop\> 
```

Comment:
Stop is now Stop/Back: pressing it again whilst stopped causes `Go Back` to be executed for the connected playback.